### PR TITLE
Adding docs on working with settings from the CLI.

### PR DESCRIPTION
**Pull request by nate-axonius**

### CODE REVIEW

File: docs/main/usage_cli/common_examples/settings_examples/ex3.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                                      AX_EXPORT_FORMAT; default: str]
+      -se, --section TEXT             Settings section internal name (not title)
+                                      [env var: AX_SECTION; required]
+      -sb, --sub-section TEXT         Settings sub section internal name (not
+                                      title)  [env var: AX_SUB_SECTION; required]
+      -c, --config SPLIT_EQUALS       Configuration keys in the form of key=value
+                                      (multiples)  [env var: AX_CONFIG]
+      --help                          Show this message and exit.
+
+An example might look like this:
```
Comment (atav-ax) [inline docs/main/usage_cli/common_examples/settings_examples/ex3.rst:64]: @nate-axonius change this to 
"Here is an example:"

File: docs/main/usage_cli/common_examples/settings_examples/ex3.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+.. code:: shell
+
+    $ axonshell system settings-global update-sub-section --section "email_settings" \
+    --config "sender_address=no-reply@yourdomain.com"
+
+To update a section or sub-section using json from a file you could use the ``update-sub-section-json`` command.
+
+.. code:: shell
+
+    $ axonshell system settings-global update-sub-section-from-json --help
```
Comment (atav-ax) [inline docs/main/usage_cli/common_examples/settings_examples/ex3.rst:75]: @nate-axonius is there an example to how the json file looks like?

**Review by atav-ax — CHANGES_REQUESTED** [review]

File: docs/main/usage_cli/common_examples/settings_examples/ex3.rst (hunk 1 above)
Comment (nate-axonius) [reply docs/main/usage_cli/common_examples/settings_examples/ex3.rst]: Fixed.

File: docs/main/usage_cli/common_examples/settings_examples/ex3.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      --help                          Show this message and exit.
+
+An example might look like this:
+
+.. code:: shell
+
+    $ axonshell system settings-global update-sub-section --section "email_settings" \
+    --config "sender_address=no-reply@yourdomain.com"
+
+To update a section or sub-section using json from a file you could use the ``update-sub-section-json`` command.
```
[the remark this answers is not in the document]
Comment (nate-axonius) [reply docs/main/usage_cli/common_examples/settings_examples/ex3.rst]: Fixed.

File: docs/main/usage_cli/common_examples/settings_examples/ex3.rst (hunk 2 above)
Comment (nate-axonius) [reply docs/main/usage_cli/common_examples/settings_examples/ex3.rst:75]: Fixed.

**Review by atav-ax — APPROVED** [review]

**Pull request merged**